server/sensors/nfc.js

```diff
@@ -1,7 +1,18 @@
 const { NFC } = require('nfc-pcsc');
 
 const listeners = [];
-const nfc = new NFC();
+
+const logger = {
```

Comment:
The logger isn't necessary any longer?

Reply:
Not for this issue, since we don't get any errors logged. I can remove it.